tests/test_journal.py

```diff
@@ -3715,5 +3714,64 @@ def test_resubmission(self, journal, openreview_client, helpers):
         assert joelle_client.get_edges(invitation='TMLR/Reviewers/-/Pending_Reviews', tail='~Peter_Snow1')[0].weight == 0
 
 
+    def test_submission_with_many_authors(self, journal, openreview_client, test_client, helpers):
+
+        test_client = OpenReviewClient(username='test@mail.com', password='1234')
+        venue_id = journal.venue_id
+        raia_client = OpenReviewClient(username='raia@mail.com', password='1234')
+        joelle_client = OpenReviewClient(username='joelle@mailseven.com', password='1234')
+        editor_in_chief_group_id = journal.get_editors_in_chief_id()
+
+        authors = ['SomeFirstName User']
+        authorids = ['~SomeFirstName_User1']
+        for i in alc:
+            for j in alc[:5]:
+                profile_client = helpers.create_user(f'author_{i}{j}@mail.com', 'Author', f'TMLR {i}{j}')
```

Comment:
Do the profiles actually need to be created? If not, it might be better not to in order to keep the test fast.

Reply:
TMLR requires all the authors to have a profile. 

there is a hint to run this single test:

```
pytest tests/test_journal.py::TestJournal::test_setup tests/test_journal.py::TestJournal::test_submission_with_many_authors
```